tests/suites/main_test.function

```diff
@@ -249,6 +257,7 @@ int main(int argc, const char *argv[])
     const char **test_files = NULL;
     int testfile_count = 0;
     int option_verbose = 0;
+    int tests_stdout;
```

Comment:
I would prefer if this was initialized to something, even though this is not strictly necessary.